routes/api/contacts.js

```diff
@@ -22,4 +23,14 @@ router.put('/:contactId', async (req, res, next) => {
   res.json({ message: 'template message' })
 })
 
+//dodanie funkcji contactsServices
+router.get('/', async (req, res, next) => {
```

Comment:
To pobiera, potrzebujesz jeszcze dodać pozostałe ścieżki: POST, PUT, DELETE i GET by id
np 
```
router.delete('/:contactId', async (req, res, next) => {
  res.json({ message: 'template message' }) // to trzeba rozwinąć
})
```